notebooks/B_CNN_Data_Preview_Images.ipynb

```diff
@@ -255,7 +255,7 @@
     "    \n",
     "    # Add counts to label counter:\n",
     "    for lab in counts:\n",
-    "        label_count[lab[0]] += lab[1]\n",
+    "        label_count[lab[0]] = lab[1]\n",
```

Comment:
I think this change may break @KristinaUlicna's code

Reply:
I have a whole new notebook to replace notebook B now (uses the cellx loader), will submit a PR with the changes soon! So don't worry about that :-)